tests/e2e/tx/benchmarks_test.go

```diff
@@ -80,7 +76,7 @@ func BenchmarkTx(b *testing.B) {
 			// - tx.* events: tx.fee, tx.acc_seq, tx.signature
 			// - Sending Amount to recipient: coin_spent, coin_received, transfer and message.sender=<val1>
 			// - Msg events: message.module=bank and message.action=/cosmos.bank.v1beta1.MsgSend (in one message)
-			assert.Equal(b, 12, len(res.GetResult().GetEvents()))
+			assert.Equal(b, 10, len(res.GetResult().GetEvents()))
```

Comment:
Which ones are not send anymore?

Reply:
compared with old main commits, observed that few of the duplicate events were removed.